torch/_ops.py

```diff
@@ -438,6 +438,12 @@ def __str__(self):
     def name(self):
         return self._name
 
+    def pure(self):
```

Comment:
Shouldn't this be on `HigherOrderOperator` instead of being on `OperatorBase`? Having it on operatorbase might be weird if we already cache operators whose `pure()` methods return False (since they are, in fact, actually pure).

Reply:
What do you mean this is on `HigherOrderOperator `?

Reply:
O im blind sorry yes carry on lol